src/js/button.js

```diff
@@ -12,6 +12,13 @@ import {assign} from './utils/obj';
  * @extends ClickableComponent
  */
 class Button extends ClickableComponent {
+  constructor(player, options) {
+    super(player, options);
+
+    this._mouseused = false;
```

Comment:
Is there a reason this property name doesn't follow other conventions? Most of the time, it's camel case with a trailing underscore (i.e. `mouseUsed_`).